app/Main.js

```diff
@@ -26,6 +26,12 @@ function runApp(initial_state) {
     console.log('Initial state', initial_state);
     const config = initial_state.offchain.config
     steem.config.set('websocket', config.ws_connection_client);
+    if (config.custom_addr_prefix) {
+        steem.config.set('address_prefix', config.custom_addr_prefix);
```

Comment:
i would recommend setting address_prefix to `STM` in defaults.json, and dropping this logic and the use of the `custom_` prefixes.  Then to use a 'custom' one you just change address_prefix.